static/userIdDefaultDataType.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "uri": "http://sinopia.io/vocabulary/literalAttribute/userIdDefault",
+    "label": "userid default"
```

Comment:
Will this be exposed to users? In which case perhaps something more like `user ID default` might be better?